models/yang/sonic/sonic-ifa.yang

```diff
@@ -0,0 +1,75 @@
+module sonic-ifa {
+    namespace "http://github.com/Azure/sonic-ifa";
+    prefix ifa;
+
+    import sonic-tam {
+        prefix stam;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC TAM IFA";
+
+    revision 2019-05-15 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-ifa {
+
+        container TAM_INT_IFA_FEATURE_TABLE {
+
+            list TAM_INT_IFA_FEATURE_TABLE_LIST {
+                key "feature";
```

Comment:
Change this to 'name' instead of 'feature'.

Can the key have any other string apart from "feature" ? If not define as enum instead of string or have pattern for string type.

Reply:
Done